integration/examples/custom-tests/command/basic_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Skaffold Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Note: This is an example from `Go by Example` site (https://gobyexample.com/testing)
```

Comment:
This code is licensed by CC-BY 3.0; I think this is ok to include but we maybe should check (https://opensource.google/docs/thirdparty/licenses/) — it's unusual to have Creative Commons on code.  To be honest, I think you should create a new example from scratch.

Reply:
Added these tests per the suggestion here - https://github.com/GoogleContainerTools/skaffold/pull/5394#discussion_r575510745

I'll revisit the sample.

Reply:
Updated the sample.